presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -43,7 +43,7 @@
 public final class LogsHandler
 {
     // TODO: Make _all_ of these 'private static final' values configurable
-    private static final Path LOG_DIRECTORY = Paths.get("/tmp", "presto", "logs");
+    private static final Path LOG_DIRECTORY = Paths.get("/var/log/presto");
```

Comment:
It's better the other way, and this line won't be sticking around for long anyway.